### DISCUSSION TITLE
Error: 'You cannot own more then 0 outposts'

### DISCUSSION THREAD

**Discussion by Atmois** [TownyAdvanced/Towny#6645] [Q&A] [answered]
As attached below I have the config to limit outposts based on town level and I have attached the town level for 1 resident (Same issue occurs for more residents and it's set as for every resident (Up to 10) you gain 1 extra outpost).

![38AFE2A1-E3A2-4741-A8D6-8DB68D6597B2](https://user-images.githubusercontent.com/130537361/235545032-113d6363-835a-4f3d-814e-98bd56ac5886.png)

![C3C0DA62-EEA9-48A9-903D-50957BD056F9](https://user-images.githubusercontent.com/130537361/235545034-5768856b-edc9-42b6-8a8d-2f18a3b4f53b.png)

```
- townBlockTypeLimits: {}
    upkeepModifier: 1.0
    townOutpostLimit: 4
    numResidents: 1
    debtCapModifier: 0
    namePostfix: ''
    bankCapModifier: 1.0
    mayorPrefix: ''
    townBlockBuyBonusLimit: 10
    namePrefix: ''
    mayorPostfix: ''
    peacefulCostMultiplier: 0
    townBlockLimit: 16
    resourceProductionModifier: 1.0

**Comment by LlmDl** [comment, marked as answer]
This should be filed as a bug report with teh requested files linked. Partial configs/screenshots doesn't give a full picture.

Edit: PowerRanks was also on the server breaking permission nodes.

**Reply by Atmois** [reply]
Alright will do.
